CHANGELOG.asciidoc

```diff
@@ -3,6 +3,44 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-9.0.1]]
+=== Beats version 9.0.1
+https://github.com/elastic/beats/compare/v9.0.0\...v9.0.1[View commits]
+
+==== Breaking changes
+
+*Metricbeat*
+
+- Handle permission errors while collecting data from Windows services and don't interrupt the overall collection by skipping affected services. {issue}40765[40765] {pull}43665[43665]
```

Comment:
Not a breaking change as David already fixed in the MD docs.

Reply:
Fixed in https://github.com/elastic/beats/pull/44172/commits/450a5fdf3801e7881c9b97b08453b39b31f9ca21